src/JackTrip.h

```diff
@@ -300,6 +300,10 @@ class JackTrip : public QObject
     {
         mDataProtocolReceiver = DataProtocolReceiver;
     }
+    virtual int getBufferStrategy()
```

Comment:
=>  int getBufferStrategy() const

Reply:
and noexcept

Reply:
...thanks, done. What should the noexcept syntax look like for this?

Reply:
int getBufferStrategy() const noexcept

Reply:
I think the correct order is cv specifier, reference, exception, (override/final)

Reply:
Would you remove the virtual? Or is there a reason for it?